runbotics/runbotics-orchestrator-ui/src/src-app/Actions/excel.actions.ts

```diff
@@ -643,6 +644,7 @@ const getExcelActions: () => Record<string, IBpmnAction> = () => ({
             formData: {},
         },
     },
+    [ExcelAction.DELETE_ROWS]: excelAction.deleteRows(),
```

Comment:
💚

Reply:
imo this function should be something like `excelAction.getDeleteRowsAction()`. "deleteRows" is an operation but we expect here an object to be returned

Reply:
true, didn't see that. All refactored actions are prefixed with getAction (because e.g. getGetCellAction would sound wrong) -> so here getActionDeleteRows